core/arch/arm/plat-nuvoton/npcm845x_utils.h

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Header file SPDX tag should use `/* ... */` inline comment syntax, not `//`.

By the way, if you really want this platform specif banner, I strongly suggest you implement this `print_version()` function locally in your platform _main.c_ file. No need for extra _.../plat-nuvoton/utils/npcm845x_utils.c_ and _plat-nuvoton/npcm845x_utils.h_ files.

Reply:
Fully agree